packages/desktop-client/src/components/autocomplete/NewAutocomplete.tsx

```diff
@@ -1,27 +1,47 @@
 import React, { useState } from 'react';
 import Select, {
+  GroupBase,
   Props as SelectProps,
   PropsValue,
   SingleValue,
   SelectInstance,
 } from 'react-select';
 
+import type { CreatableProps } from 'react-select/creatable';
 import CreatableSelect from 'react-select/creatable';
 
 import { NullComponent } from '../common';
 
 import styles from './autocomplete-styles';
 
-type OptionValue = string;
+type OptionValue = {
+  __isNew__?: boolean;
+  label: string;
+  value: true;
+};
```

Comment:
Before: `OptionValue: string`

After: `OptionValue: ...`

I previously set the incorrect interface here.. oops. Fixing it in this PR.